### DISCUSSION TITLE
Setting up mobile on new phone, can't log in, how do I find 'server name'?

### DISCUSSION THREAD

**Discussion by jamminjames** [LiveHelperChat/livehelperchat#2125] [Q&A]
I have a new android phone, and have installed the LHC app, however, I can't log in. I'm wondering if it's because I don't have the 'server name' set correctly, which the login screen asks for. But I can't find a setting for that anywhere when signed into LHC on my desktop. How do I find what the 'server name' is set as?

**Comment by jamminjames** [comment]
Update: I logged out using my old phone and logged back in successfully. I used the 'server name' from that on my new phone. I also realized what I thought was a lowercase 'l' (as in 'L') in the pw was supposed to be an uppercase 'I' (as in 'eye'). 

However, for future reference how DO you find the 'server name' you had set? I can't find it anywhere in my settings.

**Reply by remdex** [reply]
Mobile app by default has demo logins entered so follow same structure...

**Reply by jamminjames** [reply]
But if it's already been set up, and a person is in this same situation, setting up another device, how can they look up the server name they've used before?

**Reply by remdex** [reply]
Everything there
https://doc.livehelperchat.com/docs/mobile

you can also try with `index.php` at the end

**Reply by jamminjames** [reply]
![LHC_mobile_login-Screenshot](https://github.com/user-attachments/assets/036571f8-0803-4d9c-bff1-87d52e6be5ff)

I've looked there, and don't see 'server name' mentioned. The url of the login, yes, but not 'server name', which the login screen asks for (see screenshot).

The page shows this:

How to login?
Login to demo of live helper chat details:

https://demo.livehelperchat.com
Login: admin
Password: demo
Un check "Append index.php to address"

...however, that leaves out Server Name, which is above the login link on the login screen, and is a required field. The documentation ought to mention this, otherwise, it leads to confusion.

**Reply by noppie99** [reply]
i am having a hard time login into the Android app also..   I have the live helper chat loaded on the same hosting account  using the same domain name as my wp site.  I have it in a different directory.  I went to the Cpanel to get the server name and it still did not work
keeps saying error in longin.  and the link above  https://doc.livehelperchat.com/docs/mobile  does not give any information on login .

**Reply by jamminjames** [reply]
The 'server name' for me wound up being the same as the url, eg: https://www.yourdomain.com/livehelperchat/thc_web (no slash at the end), or wherever your 'index.php' file is for the app.

**Comment by noppie99** [comment]
thank you that worked.

**Reply by jamminjames** [reply]
Glad it did. Indeed, the documentation ought to mention "Server name", and what goes there. And "Server name" is misleading, it ought to say "Server url", imho.

**Comment by noppie99** [comment]
Yes I do agree, I went to my cpanel and tried to use that server name.  Yes
it is confusing, I wonder how many people download this app and never get
it working because of that.
